homeassistant/components/daikin/sensor.py

```diff
@@ -54,15 +55,15 @@ class DaikinSensorEntityDescription(SensorEntityDescription, DaikinRequiredKeysM
 SENSOR_TYPES: tuple[DaikinSensorEntityDescription, ...] = (
     DaikinSensorEntityDescription(
         key=ATTR_INSIDE_TEMPERATURE,
-        name="Inside Temperature",
+        name="Inside temperature",
```

Comment:
Shouldn't this be "Indoor temperature" to match "Outdoor temperature" below?

Reply:
I changed "outdoor" to "outside" to match `pydakin`'s property name